api/src/main/java/au/org/aodn/nrmn/restapi/config/RLSClientConfig.java

```diff
@@ -0,0 +1,21 @@
+package au.org.aodn.nrmn.restapi.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.ExchangeStrategies;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class RLSClientConfig {
+
+    @Bean
+    public WebClient rlsClient(@Value("${app.rls.api.url}") String rlsApiUrl) {
+        return WebClient.builder().baseUrl(rlsApiUrl).exchangeStrategies(ExchangeStrategies.builder()
+                .codecs(configurer -> configurer
+                        .defaultCodecs()
+                        .maxInMemorySize(20 * 1024 * 1024))
```

Comment:
If the current response in 18MB it might be better to set this to an order of magnitude greater. ie 100MB

Reply:
Yeah, I wasn't sure if I wanted to allow that much RAM to be consumed by this task but I guess it shouldn't get that high now that I am just processing raw JSON strings. I'll increase it to 100